src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
toate astea par a fi stilurile de la demo app .. tre sa ramana in proiect?

Reply:
le-am sters